conveyal/analysis-ui
# Clear project and clear displayed data

When you clear a project, clear the related isochrone / displayed data.

## How to test

1. Run a single point analysis with a comparison selected.
2. Select a destination layer.
3. Clear the project.
4. Ensure that the displayed comparison data is cleared.
